src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.cs

```diff
@@ -263,6 +263,7 @@ public void EndInit_DoesNotEnableEventRaisedEvents()
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/mono/mono/issues/17547", TestPlatforms.OSX, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
This should just be disabled using `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsPreciseGcSupported))]`

Reply:
Done